samples/splitter/splitter.cpp

```diff
@@ -448,8 +448,10 @@ void MyFrame::OnReplace(wxCommandEvent& WXUNUSED(event) )
 {
     if (m_replacewindow == NULL) {
         m_replacewindow = m_splitter->GetWindow2();
-        m_splitter->ReplaceWindow(m_replacewindow, new wxPanel(m_splitter, wxID_ANY));
-        m_replacewindow->Hide();
+        if (m_replacewindow != NULL) {
```

Comment:
```suggestion
        if ( m_replacewindow ) {
```

We can just use this cleaner syntax (which is also similar to what is done in most other places).